### ISSUE TITLE
E013 false positive?

### ISSUE THREAD

**Issue by evilebottnawi** [twbs/bootlint#276]
My code:
```html
<div class="col-xs-12 col-sm-6 col-md-4">
    Grid item
</div>
<div class="col-xs-12 col-sm-6 col-md-4">
    Grid item
</div>
<div class="clear visible-sm"></div>
<div class="col-xs-12 col-sm-6 col-md-4">
    Grid item
</div>
<div class="clear visible-md"></div>
<div class="col-xs-12 col-sm-6 col-md-4">
    Grid item
</div>
<div class="clear visible-sm"></div>
<div class="col-xs-12 col-sm-6 col-md-4">
    Grid item
</div>
<div class="col-xs-12 col-sm-6 col-md-4">
    Grid item
</div>
<div class="clear visible-sm"></div>
<div class="clear visible-md"></div>
<div class="col-xs-12 col-sm-6 col-md-4">
    Grid item
</div>
<div class="col-xs-12 col-sm-6 col-md-4">
    Grid item
</div>
<div class="clear visible-sm"></div>
<div class="col-xs-12 col-sm-6 col-md-4">
    Grid item
</div>
<div class="clear visible-md"></div>
<div class="col-xs-12 col-sm-6 col-md-4">
    Grid item
</div>
<div class="clear visible-sm"></div>
```
I get this code from: https://github.com/twbs/bootstrap/issues/9454

**Comment by cvrebert** [comment]
I assume you're referring to a "E013: Only columns (`.col-*-*`) may be children of `.row`s" error?
Your code has `clear` instead of the original code's `clearfix`.

It would be good for the error message to mention that `.clearfix`es are valid children though.

**Comment by evilebottnawi** [comment]
@cvrebert thanks
